tests/api/InclusionProofTest.ts

```diff
@@ -1,39 +1,129 @@
 import { Authenticator } from '../../src/api/Authenticator.js';
-import { InclusionProof } from '../../src/api/InclusionProof.js';
+import { InclusionProof, InclusionProofVerificationStatus } from '../../src/api/InclusionProof.js';
 import { RequestId } from '../../src/api/RequestId.js';
-import { Transaction } from '../../src/api/Transaction.js';
 import { DataHash } from '../../src/hash/DataHash.js';
 import { HashAlgorithm } from '../../src/hash/HashAlgorithm.js';
 import { SigningService } from '../../src/signing/SigningService.js';
-import { SparseMerkleTree } from '../../src/smt/SparseMerkleTree.js';
+import { MerkleTreePath } from '../../src/smt/MerkleTreePath.js';
 import { HexConverter } from '../../src/util/HexConverter.js';
 
 describe('InclusionProof', () => {
-  it('should encode and decode to exactly same object', async () => {
-    const signingService = new SigningService(
-      new Uint8Array(HexConverter.decode('0000000000000000000000000000000000000000000000000000000000000001')),
-    );
-    const transactionHash = DataHash.fromImprint(new Uint8Array(34));
-    const authenticator = await Authenticator.create(
-      signingService,
-      transactionHash,
-      DataHash.fromImprint(new Uint8Array(34)),
-    );
-    const smt = new SparseMerkleTree(HashAlgorithm.SHA256);
-    const transaction = await Transaction.create(authenticator, transactionHash);
-    const requestId = await RequestId.create(signingService.publicKey, authenticator.stateHash);
-    smt.addLeaf(requestId.toBigInt(), transaction.leafValue.imprint);
-    const inclusionProof = new InclusionProof(await smt.getPath(requestId.toBigInt()), authenticator, transactionHash);
+  const signingService = new SigningService(
+    new Uint8Array(HexConverter.decode('0000000000000000000000000000000000000000000000000000000000000001')),
+  );
+  const transactionHash = DataHash.fromImprint(new Uint8Array(34));
+  const authenticator = Authenticator.fromCBOR(
+    HexConverter.decode(
+      '8469736563703235366b3158210279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f817985841a0b37f8fba683cc68f6574cd43b39f0343a50008bf6ccea9d13231d9e7e2e1e411edc8d307254296264aebfc3dc76cd8b668373a072fd64665b50000e9fcce5201582200000000000000000000000000000000000000000000000000000000000000000000',
```

Comment:
common issue with hardcoded test data is when some functionality or data structures change, it might not be straightforward or obvious how to regenerate such hex strings. pls add comments where this data comes from, might save some headache in the future :)